### ISSUE TITLE
Frontend

### ISSUE THREAD

**Issue by ajatsoft** [thepercival/vobetting-api#3]
I am interesting in do front end app with flutter to an api, have yours deployed yet?

**Comment by thepercival** [comment]
Hello,

Could you form correct English sentences, please?

Greetings

Coen Dunnink

**Comment by ajatsoft** [comment]
Yes, sorry for my English, I was asking you if you have published the api, to see if I can make an app in dart, with flutter, to use as the front end of your api

**Comment by thepercival** [comment]
Hello,

The vobetting-api is for personal use. If you're interested in sports in
general, you could use the fctoernooiapi, API.fctoernooi.nl.

Let me know.

Greetz Coen
